en/04_Changelogs/beta/5.0.0-beta1.md

```diff
@@ -11,7 +66,86 @@ We will only consider merging post-beta breaking API changes if they are necessa
+
+- [Yubikey registration fails](https://github.com/silverstripe/silverstripe-webauthn-authenticator/issues/124)
+- [Fix broken JavaScript tests](https://github.com/silverstripe/silverstripe-admin/issues/1421)
+- [Clear up errors and warnings when running yarn commands](https://github.com/silverstripe/silverstripe-admin/issues/1416)
+- [React context no longer has "You have unsaved changes" confirm dialog](https://github.com/silverstripe/silverstripe-admin/issues/1409)
+- [Cannot delete image in WYSIWYG with delete button](https://github.com/silverstripe/silverstripe-admin/issues/1410)
+- [Console error whenever a toast pops up](https://github.com/silverstripe/silverstripe-admin/issues/1412)
+
+#### Partial build failures
+
+We have partial build failures on these modules.
+
+- silverstripe/asset-admin [![CI](https://github.com/silverstripe/silverstripe-asset-admin/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-asset-admin/actions/workflows/ci.yml)
+- silverstripe/blog [![CI](https://github.com/silverstripe/silverstripe-blog/actions/workflows/ci.yml/badge.svg?branch=4)](https://github.com/silverstripe/silverstripe-blog/actions/workflows/ci.yml)
+- silverstripe/graphql [![CI](https://github.com/silverstripe/silverstripe-graphql/actions/workflows/ci.yml/badge.svg?branch=5)](https://github.com/silverstripe/silverstripe-graphql/actions/workflows/ci.yml)
+- silverstripe/elemental-userforms [![CI](https://github.com/dnadesign/silverstripe-elemental-userforms/actions/workflows/ci.yml/badge.svg?branch=4)](https://github.com/dnadesign/silverstripe-elemental-userforms/actions/workflows/ci.yml)
+- silverstripe/versionned-admin [![CI](https://github.com/silverstripe/silverstripe-versioned-admin/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-versioned-admin/actions/workflows/ci.yml)
+- silverstripe/campaign-admin [![CI](https://github.com/silverstripe/silverstripe-campaign-admin/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-campaign-admin/actions/workflows/ci.yml)
+- silverstripe/cms [![CI](https://github.com/silverstripe/silverstripe-cms/actions/workflows/ci.yml/badge.svg?branch=5)](https://github.com/silverstripe/silverstripe-cms/actions/workflows/ci.yml)
+- silverstripe/elemental-banner-block [![CI](https://github.com/silverstripe/silverstripe-elemental-bannerblock/actions/workflows/ci.yml/badge.svg?branch=3)](https://github.com/silverstripe/silverstripe-elemental-bannerblock/actions/workflows/ci.yml)
+- silverstripe/session-manager [![CI](https://github.com/silverstripe/silverstripe-session-manager/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-session-manager/actions/workflows/ci.yml)
+- silverstripe/recipe-ccl [![CI](https://github.com/silverstripe/recipe-ccl/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/recipe-ccl/actions/workflows/ci.yml)
```

Comment:
The CCL badge shows it as passing

Reply:
I've removed that one ... my hope is that this list will be trimmed down by the time we tag the beta.